ui/redux/actions/livestream.js

```diff
@@ -58,19 +60,46 @@ const transformLivestreamData = (data: Array<any>): LivestreamInfo => {
   }, {});
 };
 
+// TODO: change this here
 const fetchLiveChannels = async (): Promise<LivestreamInfo> => {
   const response = await fetch(LIVESTREAM_LIVE_API);
   const json = await response.json();
   if (!json.data) throw new Error();
   return transformLivestreamData(json.data);
 };
 
+/**
+ * Check whether or not the channel is used, used for long polling to display live status on claim viewing page
+ * @param channelId
+ * @returns {Promise<{channelStatus: string}|{channelData: LivestreamInfo, channelStatus: string}>}
+ */
 const fetchLiveChannel = async (channelId: string): Promise<LiveChannelStatus> => {
```

Comment:
My replays are not showing.  Perhaps old replays won't show up in new API? But should probably handle null cases gracefully.

<img width="2193" alt="image" src="https://user-images.githubusercontent.com/64950861/155051667-90b1a73d-e1ae-4c74-9135-bbd7a38345f7.png">

Reply:
<img width="754" alt="image" src="https://user-images.githubusercontent.com/64950861/155052042-30ff36bb-956e-4d1e-b46c-2d722c83d0bd.png">